wear/build.gradle.kts

```diff
@@ -16,7 +15,7 @@ android {
     defaultConfig {
         applicationId = "io.homeassistant.companion.android"
         minSdk = 26
-        targetSdk = 32
+        targetSdk = libs.versions.androidSdk.target.get().toInt()
```

Comment:
this is not correct for Wear OS as we do not handle notification permissions yet

Reply:
Another reason to have it be a different variable: we might not even be able to target the same version as the main app because it doesn't exist - the main app will probably target Android 14 later this year which doesn't exist for watches.

Reply:
yes thats a good point, Wear OS 4 comes out in October (or later this year) which is based on Android 13 so 1 version behind (at least)

Reply:
(Off-topic but the Galaxy Watch 6 with Wear OS 4 is actually releasing in August)